### PR TITLE
Fixed MFTF testing for sensitive data in system configuration

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

1) Changed the API field and private key field type from "password" to "obscure"
2) API key and Private key should be added in .credentials file instead of .env file
3) Sensitive field related issue in MFTF is resolved in 2.4.3 

### Fixed Issues (if relevant)

1. N/A

### Manual testing scenarios (*)

Before:
image

After:
image

### CODE REVIEW

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml [hunk 1]
```diff
@@ -17,8 +17,7 @@
         </arguments>
         <magentoCLI command="config:set {{AdobeStockConfigDataEnabled.path}} {{enabled}}" stepKey="setModuleEnabled"/>
         <magentoCLI command="config:set {{AdobeStockConfigDataWysiwygEnabled.path}} {{wysiwygEnabled}}" stepKey="enableWYSIWYG"/>
-<!--        TODO: uncomment next lines where issue with sensitive config setting will be resolved. -->
-<!--        <magentoCLI command="config:sensitive:set {{AdobeStockConfigDataApiKey.path}} {{apiKey}}" stepKey="setModuleApiKey"/>-->
-<!--        <magentoCLI command="config:sensitive:set {{AdobeStockConfigDataPrivateKey.path}} {{privateKey}}" stepKey="setModulePrivateKey"/>-->
+        <magentoCLI command="config:set {{AdobeStockConfigDataApiKey.path}} {{apiKey}}" stepKey="setModuleApiKey"/>
```
Comment (sidolov) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml:20]: `config:set` does not work properly for values that should be encrypted in the DB and inserts them as is. This approach will not work

**Review by sidolov — CHANGES_REQUESTED** [review]

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml (hunk 1 above)
Comment (konarshankar07) [reply AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml:20]: Hello @sidolov ,

If we move the `{{apiKey}}` from .env files to .credentials as per this [docs](https://devdocs.magento.com/mftf/docs/credentials.html) then we can manage the sensitive data and value is encrypted in MFTF 2.4.3 version. You can check this [link](https://www.youtube.com/watch?v=OHu-r1FtiZE)

Thanks

Comment (sidolov) [reply AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml:20]: I missed that you also fixed `AdobeStockImageAdminUi/Test/Mftf/Data/AdobeStockConfigData.xml` file. I'll check it, thanks!

**Review by Nazar65 — DISMISSED** [review]

**Review by engcom-Golf — CHANGES_REQUESTED** [review]
Hi @konarshankar07 Seems like api keys applied incorrect, this is result of running mftf:
DeepinScreenshot_select-area_20190821130243

DeepinScreenshot_select-area_20190821130728

**Comment by konarshankar07** [comment]
Hello @engcom-Golf 
Thanks for raising the issue. I'll check and update you

**Comment by sivaschenko** [comment]
Thanks for the pull request @konarshankar07 , however `_CREDS` are only used to mask the sensitive values in logs, they do not overcome configuration encryption. As the UI approach was chosen to fill the configuration, I am closing this pull request.

**Comment by konarshankar07** [comment]
Yes, I agree with you @sivaschenko. I forgot to update you guys that this issue is regarding MFTF so they have created internal issue where they will look into this issue. Here is the https://jira.corp.magento.com/browse/MQE-1712 url where you can track this issue.

Thanks

**Pull request closed without merging**